pkgs/top-level/perl-packages.nix

```
@@ -9214,6 +9214,20 @@ let
     };
   };
 
+  FileFindRuleVCS = buildPerlPackage {
+    pname = "File-Find-Rule-VCS";
+    version = "1.08";
```

```suggestion
    version = "1.09";
```
New version released by now